python/pydantic_core/_pydantic_core.pyi

```diff
@@ -706,25 +706,36 @@ class ValidationError(ValueError):
         Returns:
             The number of errors in the validation error.
         """
-    def errors(self, *, include_url: bool = True, include_context: bool = True) -> list[ErrorDetails]:
+    def errors(
+        self, *, include_url: bool = True, include_context: bool = True, hide_input: bool = False
+    ) -> list[ErrorDetails]:
         """
         Details about each error in the validation error.
 
         Args:
             include_url: Whether to include a URL to documentation on the error each error.
             include_context: Whether to include the context of each error.
+            hide_input: Whether to hide the input value of each error.
```

Comment:
I suggest inverting this and making `include_input: bool = True` for better consistency with the other arguments.

Reply:
Agreed. I originally used `hide_input` bc that's the name of the param we use for some of the higher level `ValidationError` logic, but consistency at the function level makes a lot more sense here.

Thanks!